source/dns/dynamic-dns.rst

```diff
@@ -116,6 +116,11 @@ Amazon Route 53
 SelfHost
 ~~~~~~~~
 
+Syndicat.com
+======================
+Free DynDNS service (gnudip, HTTP, HTTPS) with any registered domain. Commercial DDNS hosting services for larger scale setups available too. (german / english)
```

Comment:
Remove the entire second sentence as it is not relevant to pfSense and is more of an ad for the service than informing users.

Capitalize "German" and "English"